paasta_tools/cli/fsm/questions.py

```diff
@@ -121,39 +120,14 @@ def get_deploy_stanza():
         {"instancename": "security-check", },
         {"instancename": "push-to-registry", },
         {"instancename": "performance-check", },
-        {"instancename": "pnw-stagea.canary", },
-        {"instancename": "pnw-stagea.main", },
-        {"instancename": "norcal-stageb.canary", },
-        {"instancename": "norcal-stageb.main", },
-        {"instancename": "norcal-devb.canary", },
-        {"instancename": "norcal-devb.main", },
-        {"instancename": "norcal-devc.canary", },
-        {"instancename": "norcal-devc.main", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.canary", },
-        {"instancename": "nova-prod.canary", "trigger_next_step_manually": True, },
-        {"instancename": "norcal-prod.main", },
-        {"instancename": "nova-prod.main", },
+        {"instancename": "dev.everything", },
```

Comment:
yelpsoa classic had this setup:

- one stage (optional canary)
- breakpoint
- prod (optional canary)
- breakpoint
- prod
- breakpoint
- everything else

This works nicely for a couple reasons:
- If you have a bad deploy you only break one of the staging environments
- Once you've "certified" your changes in production they go to the rest of the environments

Our services we've been omitting stage canary.